cli/util/convert.go

```diff
@@ -76,6 +76,16 @@ func NewCommands() []*cli.Command {
 			Usage: "Size of index file",
 			Value: 128000,
 		},
+		&cli.UintFlag{
+			Name:  "workers",
+			Usage: "Number of workers to fetch and upload blocks concurrently",
+			Value: 50,
+		},
+		&cli.UintFlag{
+			Name:  "searchers",
+			Usage: "Number of concurrent searches for blocks",
+			Value: 40,
```

Comment:
Make it 20 by default, because 40 is too harsh. We experience often search failures in mainnet:
```
2024-10-24 08:05:49.827	failed to fetch the latest missing block index from container: search of index files failed for batch with indexes from 1740000 to 1749999: failed to initiate object search: session: init session: status: code = 1024 message = connection to the RPC node has been lost
```